File: GrowAway/app/src/main/java/com/sarah/growaway/SavedFilters.java
package com.sarah.growaway;

public class SavedFilters {

        int sunlightAmount = -1;
        //boolean maintenanceAmount = false;
        int waterAmount = -1;

        public void SavedFilters() {
        this.sunlightAmount = -1;
        //this.maintenanceAmount = false;
        this.waterAmount = -1;
        }

        public void saveFilters(int sunlightAmount, int waterAmount) {
        this.sunlightAmount = sunlightAmount;
        //this.maintenanceAmount = maintenanceAmount;
        this.waterAmount = waterAmount;

        }
}

File: app/src/main/java/com/example/myapplication/SavedFilters.kt
package com.example.myapplication

object SavedFilters {

    var sunlightAmount = -1
    var maintenanceAmount = false;
    var waterAmount = -1

    fun SavedFilters() {
        this.sunlightAmount = -1
        this.maintenanceAmount = -1
        this.waterAmount = -1
    }

    fun saveFilters(sunlightAmount: Int, maintenanceAmount: Boolean, waterAmount: Int) {
        this.sunlightAmount = sunlightAmount
        this.maintenanceAmount = maintenanceAmount
        this.waterAmount = waterAmount

    }
}